### PR TITLE
Feat/node: Added mana to outputs

### LINKED ISSUE #867: Add `mana` field to outputs - nodejs
- [x] [Basic](https://github.com/iotaledger/tips-draft/blob/tip41/tips/TIP-0041/tip-0041.md#basic-output)
- [x] [NFT](https://github.com/iotaledger/tips-draft/blob/tip43/tips/TIP-0043/tip-0043.md#nft-output)
- [x] [Account](https://github.com/iotaledger/tips-draft/blob/tip42/tips/TIP-0042/tip-0042.md#account-output)

### PR DESCRIPTION
**Pull request by kwek20**
Fixes #867

### CODE REVIEW

File: bindings/nodejs/lib/types/block/output/output.ts [hunk 1]
```diff
@@ -126,8 +126,18 @@ abstract class CommonOutput extends Output /*implements ICommonOutput*/ {
...
-    constructor(amount: bigint, unlockConditions: UnlockCondition[]) {
+    /**
+     * The amount of (stored) Mana held by the output.
+     */
+    readonly mana: u64;
+
+    constructor(
+        amount: bigint,
+        unlockConditions: UnlockCondition[],
+        mana: u64,
```
Comment (DaughterOfMars) [inline bindings/nodejs/lib/types/block/output/output.ts:136]: bigint??

Comment (Thoralf-M) [reply bindings/nodejs/lib/types/block/output/output.ts:136]: It's a bigint type alias

Comment (DaughterOfMars) [reply bindings/nodejs/lib/types/block/output/output.ts:136]: No, I mean this should be a bigint too right?

Comment (kwek20) [reply bindings/nodejs/lib/types/block/output/output.ts:136]: out of scope 🦆

Comment (kwek20) [reply bindings/nodejs/lib/types/block/output/output.ts:136]: fixed in another pr

**Review by DaughterOfMars — DISMISSED** [review]

**Review by Thoralf-M — DISMISSED** [review]

File: bindings/nodejs/lib/types/block/output/output.ts (hunk 1 above)
Comment (thibault-martinez) [inline bindings/nodejs/lib/types/block/output/output.ts:136]: Can you put mana right after amount in the constructors please ? More consistent with the TIP order

**Review by thibault-martinez — APPROVED** [review]

**Review by Thoralf-M — APPROVED** [review]

**Review by qrayven — APPROVED** [review]

**Review by DaughterOfMars — APPROVED** [review]

**Pull request merged**